tensorflow_io/core/kernels/audio_kernels.cc

```diff
@@ -264,6 +346,34 @@ class AudioResampleOp : public OpKernel {
   int64 quality_;
 };
 
+class DecodeAudioOp : public DecodeAudioBaseOp {
+ public:
+  DecodeAudioOp(OpKernelConstruction *context) : DecodeAudioBaseOp(context) {}
+
+  std::unique_ptr<DecodedAudio> decode(StringPiece &data, void *config) {
```

Comment:
Also, I don't see the `decode` here? as it only does a classify?

Reply:
I does call DecodeMp3 here: https://github.com/tensorflow/io/pull/815/files/1c9995fc63e535f5c6056caf044e6da037f19414#diff-535af6f0310d54f8fb4282b7629c027bR369